oslo/torch/nn/parallel/data_parallel/zero/sharded_optim/_utils.py

```diff
@@ -16,7 +16,7 @@
 
 from torch._utils import _flatten_dense_tensors, _unflatten_dense_tensors
 import torch.distributed as dist
-from torch._six import inf
+from torch import inf
```

Comment:
https://github.com/pytorch/pytorch/blob/0f3ffaf798186103d3597d90bcbe17e27facefe8/torch/__init__.py#L1073-L1074

How about just using `math.inf`?

Reply:
math.inf looks better.

Reply:
I have checked that torch.inf == math.inf is True.